File: benchmark/template/programs/swift/adversarial/ADV-11_R.swift
import Foundation
func main() {
    print("ADV-START"); fflush(stdout)
    let n = Int64(readLine()!)!
    var xs = [Int64](repeating: 0, count: Int(n))
    xs[0] = 1
    let first = xs[0]
    print("OBS=ALLOC:\(xs.count)|FIRST:\(first)"); fflush(stdout)
    print("ADV-END"); fflush(stdout)
}
main()

File: benchmark/template/programs/cpp/adversarial/ADV-11_R.cpp
#include <cstdio>
#include <cstdint>
#include <vector>
#include <iostream>
#include <string>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::string line;
    std::getline(std::cin, line);
    std::int64_t n = std::stoll(line);
    std::vector<std::int64_t> v(n);
    v[0] = 1;
    printf("OBS=ALLOC:%lld|FIRST:%lld\n", (long long)v.size(), (long long)v[0]); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}
